### PR TITLE
Refactor of SSL verify for webdriver

### PR DESCRIPTION
**Pull request by jsmolar**

### CODE REVIEW

**Review by JaurbanRH — APPROVED** [review]

File: testsuite/ui/webdriver.py [hunk 1]
```diff
@@ -15,15 +15,13 @@
...
+    def __init__(self, source, accept_insecure_certs, remote_url=None, binary_path=None):
         self.source = source
         self.remote_url = remote_url
         self.binary_path = binary_path
         self.webdriver = None
         self.options = ChromeOptions()
-
-        if not ssl_verify:
-            self.options.set_capability("acceptInsecureCerts", True)
+        self.options.set_capability("acceptInsecureCerts", accept_insecure_certs)
```
Comment (pehala) [inline testsuite/ui/webdriver.py:24]: Why did you change the name and not simply used `self.options.set_capability("acceptInsecureCerts", not ssl_verify)`?

Comment (jsmolar) [reply testsuite/ui/webdriver.py:24]: Because I would need to do this for both ff and chrome. By doing this, we have only one place where to define the correct value (note that, bot chrome and ff are using this the same way) and it is easily readable for the user, then to think what happens with negation.
With line `accept_insecure_certs = not ssl_verify` in the parent class this negation is more obvious IMHO.

**Review by dhlavac — APPROVED** [review]
